README.md

```diff
@@ -46,6 +46,29 @@ a system CA bundle is unavailable.
 [openssl-probe]: https://github.com/alexcrichton/openssl-probe
 [webpki-roots]: https://github.com/rustls/webpki-roots
 
+## Deployment Considerations
+
+When choosing to use `rustls-platform-verifier` or another trust store option, there are important differences to consider. These
+are primarily about root certificate availability:
+
+| Backend                          | Updates                         | Roots used                                                                                            |
```

Comment:
I think another thing (either for this table, or another) is whether each option supports locally administered roots (for developer aids like mkcert, or corporate TLS inspection). That is one facet that could make webpki-roots unsuitable.